python/data/lis/layouts/README.rst

```diff
@@ -0,0 +1,48 @@
+Physical File configurations/layouts
+====================================
+
+Test files for different configurations of TapeMarks and/or padding. All files
+contain the same LIS data, which should be loaded correctly by `dlisio.lis.load`::
+
+    *.lis
+    |
+    |-> Reel Header
+        |
+        |-> Tape Header
+            |
+            |-> File Header
+            |-> File Trailer
+            |-> File Header
+            |-> File Trailer
+
+That is, a single REEL, containing a single TAPE, containing 2 Logical Files.
+Each containing only a FHLR and a FTLR. All trailer Logical Records are
+present.
+
+Note that the actual Logical Records contain dummy data and can't be parsed by
```

Comment:
To discuss: apparently this might not be a good idea. It was difficult for me by looking at the files to figure out which record is which. Surprisingly real files give more tells about what's going on because their Headers/Trailers actually contain strings and are of more-or-less expected length. At least they were easier to investigate than custom ones.
So maybe we need to make this files more human-friendly.